pkgs/applications/emulators/dolphin-emu/default.nix

```diff
@@ -58,21 +58,16 @@
 
 stdenv.mkDerivation rec {
   pname = "dolphin-emu";
-  version = "5.0-21460";
+  version = "2407";
 
   src = fetchFromGitHub {
     owner = "dolphin-emu";
     repo = "dolphin";
-    rev = "a9544510468740b77cf06ef28daaa65fe247fd32";
-    hash = "sha256-mhD7Uaqi8GzHdR7Y81TspvCnrZH2evWuWFgXMQ2c8g0=";
+    rev = "b92e354389bb7c0bd114a8631b8af110d3cb3a14";
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
I opted to hold of on that until we were sure that they publish following releases as tags. But i found https://github.com/dolphin-emu/central/blob/1960442b2902959ebb39f536b7cab701ef189dce/central/buildbot.py#L61 which indicate they likely will.